### PR TITLE
[ANCHOR-849] Add `get_transaction` rpc method

### PR DESCRIPTION
**Pull request by JiahuiWho**
As title.

### CODE REVIEW

File: openrpc/src/anchor-platform/schemas/platform_transaction.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
+{
+  "platform_transaction": {
```
Comment (philipliu) [inline openrpc/src/anchor-platform/schemas/platform_transaction.json:2]: We already have platform schemas defined by SEP because the field values differ between SEPs. Do we want to reuse that? https://github.com/philipliu/stellar-docs/blob/main/openapi/anchor-platform/schemas.yaml#L397-L652

File: static/stellar-rpc.openrpc.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
...
+      "description": "Clients can request a filtered list of events emitted by a given ledger range.\n\nStellar-RPC will support querying within a maximum 7 days of recent ledgers.\n\nNote, this could be used by the client to only prompt a refresh when there is a new ledger with relevant events. It should also be used by backend Dapp components to \"ingest\" events into their own database for querying and serving.\n\nIf making multiple requests, clients should deduplicate any events received, based on the event's unique id field. This prevents double-processing in the case of duplicate events being received.\n\nBy default stellar-rpc retains the most recent 24 hours of events.",
```
Comment (philipliu) [inline static/stellar-rpc.openrpc.json:36]: This change seems unrelated 🤔

Comment (JiahuiWho) [reply static/stellar-rpc.openrpc.json:36]: This is generated change when run `yarn rpcspec:build` according to the doc https://github.com/stellar/stellar-docs/blob/main/openrpc/README.md#tldr

File: openrpc/src/anchor-platform/schemas/platform_transaction.json (hunk 1 above)
Comment (philipliu) [reply openrpc/src/anchor-platform/schemas/platform_transaction.json:2]: Hm, it looks like these work for openrpc. That's fine then.

File: openrpc/src/anchor-platform/schemas/platform_transaction.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      },
+      "updated_at": {
+        "$ref": "#/components/schemas/updated_at"
+      },
+      "completed_at": {
+        "$ref": "#/components/schemas/completed_at"
+      },
+      "transfer_received_at": {
+        "$ref": "#/components/schemas/transfer_received_at"
+      },
```
Comment (philipliu) [inline openrpc/src/anchor-platform/schemas/platform_transaction.json:45]: `user_action_required_by` is missing

File: openrpc/src/anchor-platform/schemas/platform_transaction.json [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+      "sep": {
+        "$ref": "#/components/schemas/sep"
+      },
+      "kind": {
+        "$ref": "#/components/schemas/kind"
+      },
+      "status": {
+        "$ref": "#/components/schemas/status"
+      },
+      "amount_expected": {
```
Comment (philipliu) [inline openrpc/src/anchor-platform/schemas/platform_transaction.json:19]: `type` is missing

File: openrpc/src/anchor-platform/schemas/platform_transaction.json [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+      "client_domain": {
+        "description": "The domain of the client.",
+        "type": "string"
+      },
+      "client_name": {
+        "description": "The domain of the client.",
+        "type": "string"
+      },
+      "customers": {
+        "description": "The Identification info of the sending and receiving customers. If they were created through [SEP-12](https://stellar.org/protocol/sep-12),\n            this object should contain the SEP-12 customer `id`. Otherwise, the `account` address of the customer.",
```
Comment (philipliu) [inline openrpc/src/anchor-platform/schemas/platform_transaction.json:89]: nit: Can this be moved to `customer.json`?

File: openrpc/src/anchor-platform/schemas/platform_transaction.json (hunk 4 above)
Comment (JiahuiWho) [reply openrpc/src/anchor-platform/schemas/platform_transaction.json:19]: They are all included in the reference object?

**Comment by JiahuiWho** [comment]
I also pinged @ElliotFriend to help take a look see if we can reuse the schema defined under openapi.

**Review by lijamie98 — APPROVED** [review]
LGTM
